bufferserver/src/test/java/com/datatorrent/bufferserver/util/CodecTest.java

```diff
@@ -22,8 +22,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.annotations.Test;
-import static org.testng.Assert.*;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
We do allow static import and use it in other classes (mostly in tests).

Bufferserver uses testng JUnit provider and it should be part of another exercise to migrate Bufferserver to org.junit from org.testng.

I will fix static import to org.testng.Assert.assertEquals.